js/app.js

```diff
@@ -363,16 +349,17 @@
 				});
 			}
 		},
+		addParticipantToCall: function(token, participant) {
+			return this.signaling.addParticipantToCall(token, participant);
```

Comment:
Ah ok, if it's the invitation then it shouldn't go through the signaling. I'll change it back to go to the PHP backend.